tests/api2/test_262_iscsi_alua.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+#
+# test_261_iscsi_cmd contains some general ALUA tests, but this file will contain some
+# more detailed ALUA tests
+import contextlib
+import random
+import string
+from time import sleep
+
+import pytest
+from assets.websocket.iscsi import (alua_enabled, initiator_portal, target,
+                                    target_extent_associate, verify_capacity,
+                                    verify_luns)
+from assets.websocket.service import ensure_service_enabled
+from middlewared.test.integration.utils import call
+from middlewared.test.integration.utils.client import truenas_server
+
+from auto_config import ha, pool_name
+from protocols import iscsi_scsi_connection
+
+if not ha:
+    pytest.skip("skipping ALUA tests", allow_module_level=True)
```

Comment:
Instead of checking `if not ha:` you can do `pytestmark = pytest.mark.skipif(not ha, reason='reason here')`